From f3c40c86c7c8c6c7865fef765a68d426a44f562b Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Sun, 6 Jul 2014 11:13:02 +1000
Subject: [PATCH 4/5] Coverity CID 1224483

If pginfo was null, the line above would have crashed. pginfo is never null
---
 mythtv/programs/mythtranscode/main.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/programs/mythtranscode/main.cpp b/mythtv/programs/mythtranscode/main.cpp
index 9cf574b..f49c66b 100644
--- a/mythtv/programs/mythtranscode/main.cpp
+++ b/mythtv/programs/mythtranscode/main.cpp
@@ -737,7 +737,7 @@ static int transUnlink(QString filename, ProgramInfo *pginfo)
 {
     QString hostname = pginfo->GetHostname();
 
-    if (pginfo != NULL && !pginfo->GetStorageGroup().isEmpty() &&
+    if (!pginfo->GetStorageGroup().isEmpty() &&
         !hostname.isEmpty())
     {
         QString ip = gCoreContext->GetBackendServerIP(hostname);
-- 
1.7.10.2

